README.md

```diff
@@ -107,171 +114,173 @@ Server running... press ctrl-c to stop.
 
-- Refer [here](https://docs.docker.com/compose/install/) for the official installation instructions.
+#### Install Ruby extension
 
-### Execution Steps
+If you have Ruby installed on the workstation, then you can skip this installation.
 
-1. Using [git](https://git-scm.com/), [clone](https://help.github.com/articles/cloning-a-repository/) this repository.
-2. Navigate to the resulting directory.
-3. Run `docker-compose up` to initialize the build process. Refer [here](https://docs.docker.com/compose/gettingstarted/#step-4-build-and-run-your-app-with-compose) for more details on the use of `docker-compose`.
-4. Visit `http://localhost:4000/` in a web browser, and you should be presented with a local copy of the Magento DevDocs. The configuration for the local port (`4000` by default) is found in the [docker-compose.yml](https://github.com/magento/devdocs/blob/develop/docker-compose.yml) file. If another port is desired, please refer [here](https://docs.docker.com/compose/compose-file/compose-file-v2/#ports) for further details regarding Docker Compose port mapping.
+1.  Open the **Command Prompt** using **Run as Administrator** in the shortcut menu.
 
-### Addressing Problems With Docker Build
+1.  Install the ruby extension:
 
-1. Verify that the Docker engine is installed for the appropriate operating system.
-2. Verify that Docker Compose is installed.
-3. Verify that this repository has been cloned.
-4. Verify that the correct Docker Compose command(s) have been used in the same directory as the `docker-compose.yml` file.
-5. If there are still problems, please open an [Issue](https://help.github.com/articles/creating-an-issue/) on this repository.
+    ```cmd
+    choco install ruby
+    ```
 
-## Build using Vagrant
+1.  Verify the environment variables were added properly:
 
-You can deploy the devdocs site locally using [this Vagrant project](https://github.com/magento-devdocs/vagrant-for-magento-devdocs).
+    -  In the Windows UI, open search and type `path`.
+    -  In the Windows CMD console, type `echo %path%`.
 
-***
+>  **NOTE**  
+>  If you encounter problems with Ruby, or the `gem` command is not recognized, you can install the `rubyinstaller-devkit.exe` development kit located in the `c:\ProgramData\chocolatey\bin` folder.
 
-If you have questions, open an issue and ask us. We're looking forward to hearing from you!
+### Install Git for Windows
 
-- [Follow @MagentoDevDocs](https://twitter.com/MagentoDevDocs)
-- [E-mail us](mailto:DL-Magento-Doc-Feedback@magento.com)
-- [Visit our documentation site](https://devdocs.magento.com), built using [GitHub pages](https://pages.github.com/).
+Use Git for Windows to prevent interference with the existing Windows environment and to have _Git Bash_ and _Git GUI_ launch commands available on the shortcut menu.
```

Comment:
```suggestion
Use [Git for Windows](https://gitforwindows.org/) to prevent interference with the existing Windows environment and to have _Git Bash_ and _Git GUI_ launch commands available on the shortcut menu.
```